lib/PubSub.jsx

```diff
@@ -72,8 +72,9 @@ const PubSubModule = {
      * @returns {String}
      */
     once(eventName, callback, optionalScope) {
-        const scope = _.isObject(optionalScope) ? optionalScope : window;
-        const functionToCallOnce = _.once((...args) => callback.apply(scope, args));
+        const scope = typeof optionalScope === 'object' && optionalScope !== null ? optionalScope : window;
+        // TODO: Check how to replace once function
```

Comment:
Any update on this TODO?

Reply:
I think for now we should just migrate `underscore` to `lodash` function